utilities/calculateFinalPot.ts

```diff
@@ -0,0 +1,36 @@
+import GameSituation from "../models/GameSituation";
+
+export default function calculateFinalPot(
+  gameSituation: GameSituation
+): number {
+  const preflopPot: number[] = new Array(gameSituation.playerLength).fill(0);
+  gameSituation.preflop.actions.forEach(({ playerIndex, betSize }) => {
+    if (betSize !== 0) preflopPot[playerIndex] = betSize;
+  });
```

Comment:
Probably you can use just `for-of`.

```suggestion
  for (const { playerIndex, betSize } of gameSituation.preflop.actions) {
    if (betSize === 0) continue;

    preflopPot[playerIndex] = betSize;
  }
```